modules/restore-regenerate-etcd-certs.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-regenerate-etcd-certs_{context}"]
+= Generating etcd certificates and adding the member to the cluster
+
+If the node is new or the etcd certificates on the node are no longer valid, you must generate the etcd certificates before you can add the member to the etcd cluster.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have SSH access to the master host to add to the etcd cluster.
```

Comment:
@retroflexer Can you double check these prereqs I added?

Reply:
Can we say `new master host` to be clear?